### PR TITLE
Npm ci

### PR DESCRIPTION
**Pull request by chrismiceli**
I also renamed some things referencing travis.

### CODE REVIEW

**Comment by chrismiceli** [comment]
I only did one commit, the rest are from @cpeel

File: .github/workflows/static-checks.yml [hunk 1]
```diff
@@ -63,3 +63,7 @@ jobs:
         run: cd SETUP && make lint_charsuites && cd ..
       - name: Run less/CSS checks
         run: cd SETUP && make lint_css && cd ..
+      - name: Run js lint
```
Comment (cpeel) [inline .github/workflows/static-checks.yml:66]: JS linting should be in the `linting` job above.

File: .github/workflows/static-checks.yml [hunk 2]
```diff
@@ -63,3 +63,7 @@ jobs:
...
+        run: npm run lint
+      - name: Run js unit tests
```
Comment (cpeel) [inline .github/workflows/static-checks.yml:68]: This should be in a separate `junit` job, similar to how `phpunit` is.

File: SETUP/tests/README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ Including:
 * [phpunit](https://phpunit.de/) tests in this directory
 * [QUnit](https://qunitjs.com/) tests in the `jsTests` directory
 * Manual web-based tests in the `manual_web` directory
-* [TravisCI](https://travis-ci.com/) configuration
+ * [GitHub Actions](https://github.com/features/actions) configuration
```
Comment (cpeel) [inline SETUP/tests/README.md:8]: nit: can you please remove this errant space?

**Review by cpeel — COMMENTED** [review]
This is *excellent*. I appreciate you cleaning up the remaining references to Travis 👍

**Pull request merged**